src/main/resources/data/minecolonies/recipes/blockminecoloniesrack.json

```diff
@@ -0,0 +1,14 @@
+{
+  "type": "minecraft:crafting_shapeless",
+  "ingredients":
+  [
+    {
+      "tag": "forge:chests"
```

Comment:
What's the default recipe for these shelves? A shapeless recipe with just a chest might be incompatible with other mods. Also don't these have more inventory space than 18 slots?

Reply:
The default recipe is 6 planks, 2 iron nuggets, and the builder wand. I updated the recipe to match the default minus the two iron nuggets.